src/main/java/uk/gov/crowncommercial/dts/scale/service/agreements/model/dto/EventType.java

```diff
@@ -12,4 +12,6 @@
 
   private String description;
 
+  private  Boolean isPreMarketEvent;
```

Comment:
I think the property name should be `preMarketActivity` - see https://github.com/Crown-Commercial-Service/ccs-scale-api-definitions/pull/81/files#diff-046ce7653c0a5c01235e166008bc0cc5edcc7f00152be4dadc4cd264b1f81ba7R791

Reply:
Done